docs/onboarding_to_cluster.md

```diff
@@ -43,46 +43,41 @@ bash scripts/onboarding.sh NAMESPACE_NAME OWNER_TEAM OPTIONAL_PROJECT_DESCRIPTIO
-- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/groups/$OWNER_TEAM`
+- A namespace in `cluster-scope/base/core/namespaces/$NAMESPACE_NAME`
+- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM`
 - An RBAC component for the project admin role `RoleBinding` in `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM` and maps it to the newly added namespace.
 
-### Enabling namespace deployment to a specific cluster
+### Enabling namespace deployment to Zero cluster
 
 ```sh
-cd cluster-scope/overlays/TARGET_CLUSTER
-kustomize edit add resource ../../base/namespaces/$NAMESPACE
+cd cluster-scope/overlays/$ENV/$TARGET_CLUSTER
+kustomize edit add resource ../../base/core/namespaces/$NAMESPACE
 ```
 
 ### Authenticate via OpenShift
-Each supported component provides a login button which states "Login via OpenShift" or "Sign in with OpenShift". This will lead the user to an authentication provider selection screen:
+Users are automatically created upon login to the appropriate cluster. This is done by logging in via SSO (any gmail account will work).
+
+For the `MOC/Zero` you can access the login from [this link](https://console-openshift-console.apps.zero.massopen.cloud/).
+
+This will lead the user to an authentication provider selection screen:
 
 ![Auth provider selection screen in OpenShift](assets/images/openshift-login.png)
 
 Please select `moc-sso` provider. Then choose the final account provider that fits you the most:
 
 ![MOC auth provider selection screen](assets/images/moc-login.png)
 
-## Giving Users Rbac Permissions
-
-The following steps enable users to access designated cluster/namespaces. This simply consists of adding such users to the appropriate OpenShift groups with access to those cluster/namespaces. Depending on the cluster, these groups may need to be encrypted, or may be stored as plaintext, each scenario requires a different set of steps.
+With the user now created, we will need to provide them with appropriate rbac access to this namespace.
 
-To recognise which scenario applies to you, please select target cluster in `cluster-scope/overlays/TARGET_CLUSTER`. You either see a `secret-generator.yaml` (encrypted users) file or `group-user_patch.yaml` file (plaintext) in this folder.
-
-Please follow the steps according to your use case:
-
-- [Creating a group or adding users as an encrypted resource](#encrypted-overlay-patch-for-users)
-- [Creating a group or adding users without encryption](#plaintext-overlay-patch-for-users)
-
-### Encrypted overlay patch for users
+## Giving Users Rbac Permissions
 
-We'll use sops/ksops for the encryption in here.
+The following steps enable users to access designated cluster/namespaces. This simply consists of adding such users to the appropriate OpenShift groups with access to those cluster/namespaces.
```

Comment:
In documentation we generally want to avoid declaring something to be "simple" or "obvious", etc. Maybe:

> The following steps enable users to access designated cluster/namespaces. This consists of adding users to the appropriate OpenShift groups.

Reply:
updated